mesonbuild/scripts/coverage.py

```diff
@@ -58,19 +63,29 @@ def coverage(outputs, source_root, subproject_root, build_root, log_dir):
             initial_tracefile = covinfo + '.initial'
             run_tracefile = covinfo + '.run'
             raw_tracefile = covinfo + '.raw'
+            if use_llvm_cov:
+                # Create a shim to allow using llvm-cov as a gcov tool.
+                llvm_cov_sh = os.path.join(log_dir, 'llvm-cov.sh')
+                with open(llvm_cov_sh, 'w') as llvm_cov_shim:
+                    llvm_cov_shim.write('#!/usr/bin/env sh\nexec {} gcov "$@"'.format(llvm_cov_exe))
+                os.chmod(llvm_cov_sh, os.stat(llvm_cov_sh).st_mode | stat.S_IEXEC)
+                gcov_tool_args = ['--gcov-tool', llvm_cov_sh]
+            else:
+                gcov_tool_args = []
```

Comment:
llvm-cov works on windows, so we can't really rely on a shell script, if we have to wrap it we need to wrap it in a python script

Reply:
I'm concerned about Windows not being able to directly execute a python script. Perhaps a bat is better in that case